initfiles/examples/embed/couchbase-simple2.ecl

```diff
@@ -0,0 +1,140 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2017 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=embedded
+//class=3rdparty
+
+IMPORT couchbase;
+IMPORT STD;
+
+/***********************************************************************************************
+   This set of example queries are based on a device oriented document structure described below
+   These samples queries rely on a bucket named 'iot'. In order to run these queries, please
+      create the iot bucket on the target couchbase server
+************************************************************************************************/
+
+server := '10.176.152.227';
```

Comment:
Yes, it should. This test should run in any machine inside and outside.  The better solution would be this:
```
server := '.' : stored('ServerIP');
```
or
```
server := '127.0.0.1' : stored('ServerIP');
```
In this case the default is the localhost, but if user wants to execute this test with a Couchbase server on a different IP address, then they can do it with -X parameter in this command:
```
./ecl-test query -X ServerIP=<couchbase_server_ipaddress> couchbase-simple2.ecl
```
with the IP above:
```
./ecl-test query -X ServerIP=10.176.152.227 couchbase-simple2.ecl
```